README.md

```diff
@@ -121,6 +121,7 @@ The above example uses app-level credentials i.e., publish profile file for depl
 
 Follow the steps to configure the secret:
   * Download the publish profile for the WebApp from the portal (Get Publish profile option)
+  * While deploying to slot, download the publish profile for slot
```

Comment:
While deploying to slot, download the publish profile for slot. Also specify the slot-name field with the name of the slot.